zwave_js_server/firmware.py

```diff
@@ -0,0 +1,70 @@
+"""Firmware update helper."""
+from typing import Any
+
+import aiohttp
+
+from .const import MAX_SERVER_SCHEMA_VERSION
+from .model.node import Node
+from .util.helpers import convert_bytes_to_base64
+
+
+async def begin_firmware_update_guess_format(
+    url: str, node: Node, filename: str, file: bytes, session: aiohttp.ClientSession
+) -> Any:
+    """Send beginFirmwareUpdate command to Node (file format to be guessed)."""
+    client = await session.ws_connect(url)
```

Comment:
Why are we not using the client class? Eg like we do in the main module.

Reply:
I made this change based on this comment: https://github.com/zwave-js/zwave-js-server/pull/213#discussion_r629456314

This is easy enough to revert if we feel it's unnecessary

CC @balloob

Reply:
What part of that comment do you mean? My point was that we could still use the Client class when opening a new connection, I think.

Reply:
Not, sure. I haven't looked at the details recently. There might be a good reason.

Reply:
Probably because we wanted to catch the version info as part of the dump.